### PR TITLE
MM mortality modeling strategy update

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Review by yongqx — APPROVED** [review]

**Review by mattkappel — APPROVED** [review]

File: docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst [hunk 1]
```diff
@@ -322,10 +330,51 @@ data_dir = J:/Project/simulation_science/multiple_myeloma/data/cause_model_input
...
+simulants in any of the multiple myeloma cause model states other than susceptible
+should be recorded as deaths due to multiple myeloma. While deaths due to other
+causes are typically modeled in Vivarium cause models among simulants with a given
+cause, simulants in multiple myeloma cause model states other than the susceptible 
+state should have zero probability of death due to other causes. Simulants without
+multiple myeloma (in the susceptible cause model state) should die due to causes
+other than multiple myeloma ("other causes") at a rate equal to the multiple
+myeloma-deleted all cause mortality rate. Details are shown in the table below.
+
+.. list-table:: MM State-Stpecitfic Mortality Hazard Rates and Causes of Death
```
Comment (mattkappel) [inline docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst:358]: s/Stpecitfic/Specific

**Comment by yongqx** [comment]
@mattkappel I am going to merge this PR, have you had change to incorporate this mortality component update into the cause model? @plinds tag you here in case we don't have a JARA ticket to track this task.

**Comment by mattkappel** [comment]
> @mattkappel I am going to merge this PR, have you had change to incorporate this mortality component update into the cause model? if not, let's hold it. @plinds tag you here in case we don't have a JARA ticket to track this task, low priority.

We do have JIRA ticket for this, and I'm running with this change now, expecting results in a few hours.

File: docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst [hunk 2]
```diff
@@ -322,10 +330,51 @@ data_dir = J:/Project/simulation_science/multiple_myeloma/data/cause_model_input
...
+   * - csmr_c486
+     - GBD 2019
+     - deaths_c486 / population
+
+Multiple Myeloma Mortality Details
++++++++++++++++++++++++++++++++++++
+
+As previously mentioned, the excess mortality rates defined in the tables above
+represent mortality rates represent *all-cause* mortality rates among patients 
+with multiple myeloma. For simplicity, in our simulation, deaths that occur among
```
Comment (KjellSwedin) [inline docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst:348]: represent mortality rates represent
extra "represent"

**Review by KjellSwedin — APPROVED** [review]

**Pull request merged**